### PR TITLE
Adjust log level and some code styling

### PR DESCRIPTION
**Pull request by TinchoIgnacio**
# Changes done
- Change some log entries from Info to Debug
- Some code styling changes

### CODE REVIEW

File: src/Splitio-net-core/Services/Cache/Classes/InMemoryReadinessGatesCache.cs [hunk 1]
```diff
@@ -36,11 +36,11 @@ public void SplitsAreReady()
                 splitsAreReady.Signal();
                 if (splitsAreReady.IsSet)
                 {
-                    Log.Info("Splits are ready");
+                    Log.Debug("Splits are ready");
```
Comment (patricioe) [inline src/Splitio-net-core/Services/Cache/Classes/InMemoryReadinessGatesCache.cs]: is it possible to modify this to: `Split are ready in xxx millis` ?

File: src/Splitio-net-core/Services/Cache/Classes/InMemoryReadinessGatesCache.cs [hunk 2]
```diff
@@ -55,7 +55,7 @@ public void SegmentIsReady(string segmentName)
 
             if (countDown.IsSet)
             {
-                Log.Info(segmentName + " segment is ready");
+                Log.Debug(segmentName + " segment is ready");
```
Comment (patricioe) [inline src/Splitio-net-core/Services/Cache/Classes/InMemoryReadinessGatesCache.cs]: same as before. It is better for debugging.

File: src/Splitio-net-core/Services/Cache/Classes/InMemoryReadinessGatesCache.cs [hunk 3]
```diff
@@ -112,7 +112,7 @@ public bool AreSegmentsReady(int milliseconds)
                 timeLeft = timeLeft - (int)clock.ElapsedMilliseconds;
             }
 
-            Log.Info("Segments are ready");
+            Log.Debug("Segments are ready");
```
Comment (patricioe) [inline src/Splitio-net-core/Services/Cache/Classes/InMemoryReadinessGatesCache.cs]: same

File: src/Splitio-net-core/Services/SegmentFetcher/Classes/SegmentTaskWorker.cs [hunk 4]
```diff
@@ -41,11 +42,11 @@ public void ExecuteTasks(CancellationToken token)
                     //Wait indefinitely until a segment is queued
                     if (SegmentTaskQueue.segmentsQueue.TryTake(out segment, -1))
                     {
-                        Log.Info(string.Format("Segment dequeued: {0}", segment.name));
+                        Log.Debug(string.Format("Segment dequeued: {0}", segment.name));
```
Comment (patricioe) [inline src/Splitio-net-core/Services/SegmentFetcher/Classes/SegmentTaskWorker.cs]: is there in .Net something like:

``` 
if(Log.isDebugEnabled()) {
Log.Debug(string.Format("Segment dequeued: {0}", segment.name)) 
}```

Why?

Usually, at least in java, the VM contractus all the objects to pass as params to the method, then enters the method to find you are NOT in debug mode, so nothing gets logged.

File: src/Splitio-net-core/Services/SegmentFetcher/Classes/SelfRefreshingSegment.cs [hunk 5]
```diff
@@ -51,23 +51,23 @@ public SelfRefreshingSegment(string name, ISegmentChangeFetcher segmentChangeFet
 
                         if (response.added.Count() > 0)
                         {
-                            Log.Info(string.Format("Segment {0} - Added : {1}", name, string.Join(" - ", response.added)));
+                            Log.Debug(string.Format("Segment {0} - Added : {1}", name, string.Join(" - ", response.added)));
```
Comment (patricioe) [inline src/Splitio-net-core/Services/SegmentFetcher/Classes/SelfRefreshingSegment.cs]: same

File: src/Splitio-net-core/Services/SegmentFetcher/Classes/SelfRefreshingSegment.cs [hunk 6]
```diff
@@ -51,23 +51,23 @@ public SelfRefreshingSegment(string name, ISegmentChangeFetcher segmentChangeFet
...
                         }
                         if (response.removed.Count() > 0)
                         {
-                            Log.Info(string.Format("Segment {0} - Removed : {1}", name, string.Join(" - ", response.removed)));
+                            Log.Debug(string.Format("Segment {0} - Removed : {1}", name, string.Join(" - ", response.removed)));
```
Comment (patricioe) [inline src/Splitio-net-core/Services/SegmentFetcher/Classes/SelfRefreshingSegment.cs]: same

File: src/Splitio-net-core/Services/SegmentFetcher/Classes/SelfRefreshingSegment.cs [hunk 7]
```diff
@@ -51,23 +51,23 @@ public SelfRefreshingSegment(string name, ISegmentChangeFetcher segmentChangeFet
...
+                    segmentCache.SetChangeNumber(name, response.till);
                 }
                 catch (Exception e)
                 {
                     Log.Error("Exception caught refreshing segment", e);
                 }
                 finally
                 {
-                    Log.Info(string.Format("segment {0} fetch before: {1}, after: {2}", name, changeNumber, segmentCache.GetChangeNumber(name)));
+                    Log.Debug(string.Format("segment {0} fetch before: {1}, after: {2}", name, changeNumber, segmentCache.GetChangeNumber(name)));
```
Comment (patricioe) [inline src/Splitio-net-core/Services/SegmentFetcher/Classes/SelfRefreshingSegment.cs]: same

File: src/Splitio-net-core/Services/SplitFetcher/Classes/SelfRefreshingSplitFetcher.cs [hunk 8]
```diff
@@ -123,7 +122,7 @@ private void UpdateSplitsFromChangeFetcherResponse(List<Split> splitChanges)
                 }
                 finally
                 {
-                    Log.Info(string.Format("split fetch before: {0}, after: {1}", changeNumber, splitCache.GetChangeNumber()));
+                    Log.Debug(string.Format("split fetch before: {0}, after: {1}", changeNumber, splitCache.GetChangeNumber()));
```
Comment (patricioe) [inline src/Splitio-net-core/Services/SplitFetcher/Classes/SelfRefreshingSplitFetcher.cs]: same

File: src/Splitio-net-core/Services/SegmentFetcher/Classes/SegmentTaskWorker.cs (hunk 4 above)
Comment (TinchoIgnacio) [reply src/Splitio-net-core/Services/SegmentFetcher/Classes/SegmentTaskWorker.cs]: Yes there is. The thing is that building the log message is expensive only if the ToString of an object that we are logging is expensive (which is not the case for the entries we are logging). 
Anyway, if we invoke `Log.Debug` only if `Log.isDebugEnabled()` we are doing no harm so I will include it

File: src/Splitio-net-core/Services/Cache/Classes/InMemoryReadinessGatesCache.cs (hunk 1 above)
Comment (TinchoIgnacio) [reply src/Splitio-net-core/Services/Cache/Classes/InMemoryReadinessGatesCache.cs]: Sure, I can do some changes to the class in order to log the time it took. I can start a 2 timers when the class gets constructed and then check them for logging

**Pull request merged**